packages/frontend/src/components/ui/kpi-token-card/time-left/index.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { Timer, Typography } from "@carrot-kpi/ui";
+import { ResolvedKPIToken } from "@carrot-kpi/sdk";
+import { useTranslation } from "react-i18next";
+
+interface TimeLeftProps {
+    kpiToken: ResolvedKPIToken;
+}
+
+export const TimeLeft = ({ kpiToken }: TimeLeftProps) => {
+    const { t } = useTranslation();
+
+    if (kpiToken.finalized)
+        return (
+            <Typography uppercase className={{ root: "text-orange" }}>
+                {t("stateOptions.finalized")}
+            </Typography>
+        );
+    if (kpiToken.expired)
+        return (
+            <Typography uppercase className={{ root: "text-red" }}>
+                {t("stateOptions.expired")}
+            </Typography>
+        );
+    return <Timer to={kpiToken.expiration} countdown />;
```

Comment:
Should we also show the seconds here? Or at least expect a `seconds` prop for `TimeLeft` that will be provided to <Timer>?

Reply:
The countdown in the card once showed the seconds, but then I removed it because it made the text break and go to the next line, which was ugly. Hence why we now only show minutes.